src/Compat/CompatForm.php

```diff
@@ -61,6 +61,14 @@ public function duplicateSubmitButton(FormSubmitElement $originalSubmitButton):
             ->set('class', 'primary-submit-btn-duplicate');
         $attributes->remove('id');
 
-        return new SubmitElement($originalSubmitButton->getName(), $attributes);
+        $newSubmitButton = new SubmitElement($originalSubmitButton->getName(), $attributes);
+
+        if ($originalSubmitButton instanceof SubmitElement) {
+            $newSubmitButton->setLabel($originalSubmitButton->getButtonLabel());
+        } else {
+            $newSubmitButton->setLabel($originalSubmitButton->getLabel());
```

Comment:
The label of SubmitButtonElements have no purpose in this regard, they exist only for display reasons. It's the submit value that needs to be kept in this case.